### ISSUE TITLE
Show when mini ftool is activated(the key was pressed) and when is not activated, also add which loop number are we in.

### ISSUE THREAD

**Issue by familypc123** [ils94/PyFlyff#23]
Show when mini ftool is activated(the key was pressed) and when is not activated, also add which loop number are we in.
I mean i don't know when will it stop.

**Comment by direrFlood** [comment]
I think he means the number of loop. (loop number)/Repeats.
yes this would be a cool feature

**Comment by ils94** [comment]
the loop will always be from the first to the second and third key even if you stop it and resume, it will always start and restart from the first key. It easy to spot if the mini ftool is activated, if the keys are being pressed without u pressing them  it means that something is on xD

**Comment by ils94** [comment]
> > the loop will always be from the first to the second and third key even if you stop it and resume, it will always start and restart from the first key. It easy to spot if the mini ftool is activated, if the keys are being pressed without u pressing them it means that something is on xD
> > Oh so it is not separates loops for separate keys?
> > like
> > In-Game Hotkey(s): x,y,z Interval(s): 2,3,5
> > parallel do this:
> > press x -> wait 2 seconds -> press x
> > in parallel: press y -> wait 3 seconds -> press y
> > in parallel: while press z -> wait 5 seconds -> press z

Nah, would be kinda insane to have a loop separated for each key, even more that since the client create a separated thread to execute the mini ftool loop, having to watch them all to know when to use a key is kinda too much work. There is even an option to fix rotation, it there for a reason, because doesn't matter how you configure the loop to press the keys in a sequence, the sequence will break, meaning that there will be a time where the loop will finally press 2 or 3 keys at the sometime because they will mathematically reach a point where all 3 buttons reached the same threshold.

Everytime the first key is pressed, it is added into a variable that is checked to see if the time passed is the time to press the second key, same for the third key.
